### PR TITLE
Revised from previous PR89: Add Encryption function options

### PR DESCRIPTION
**Pull request by kodeschreiber**
- Added the "sqlite3_key_v2" and "sqlite3_rekey_v2" functions
- Added a feature gate called "encryption" to disable the feature if not used

### CODE REVIEW

File: Cargo.toml.orig [hunk 1]
```diff
@@ -0,0 +1,43 @@
+[package]
```
Comment (IvanUkhov) [inline Cargo.toml.orig:1]: Please remove all extra files.

File: Cargo.toml [hunk 2]
```diff
@@ -33,6 +33,7 @@ keywords = ["database"]
 default = ["linkage"]
 bundled = ["sqlite3-sys/bundled"]
 extension = []
+encryption = []
```
Comment (IvanUkhov) [inline Cargo.toml:36]: Have you tried it? It should not work, as one has to enable `sqlite3-sys/encryption`.

File: Cargo.toml.orig (hunk 1 above)
Comment (kodeschreiber) [reply Cargo.toml.orig:1]: I though it was strange, that when I created this PR, those "orig" files appeared. I'm not entirely why those where generated, but I'll remove them.

File: Cargo.toml (hunk 2 above)
Comment (kodeschreiber) [reply Cargo.toml:36]: When I tried with with a test package, it does seem to react to it's absence. Admittedly, this is my first time seeing the Rust "-sys" pattern.

Comment (IvanUkhov) [reply Cargo.toml:36]: OK, I will add it myself.

**Comment by IvanUkhov** [comment]
Thank you! The same question here about the authors list. Do you want to be added, and if so how?

**Comment by kodeschreiber** [comment]
As with the other, I'm good with my username being put in the list. If not, I'm  just happy to contribute to the project.

**Pull request merged**